handouts/2020-02-04-diversity-blurb.txt

```diff
@@ -0,0 +1,22 @@
+
+There is no such thing as a meritocracy. An unchecked imbalance of power will crystallize to become a structure. Power is weaponized in the duality of privilege and discrimination but all three are invisible. If deep, structural power imbalances exist we have gone off course. We have gone off course.
+
+Structural casteism exists, obviously. The adjective is redundant. [1] Structural racism exists, insidiously. We segregate with the tone of our voice, body language, and unwritten rules that dictate who belongs (and who does not). [2,3] Structural sexism exists, inside your house. How to see a structure built over thousands of years, woven into the fabric of society? [4] To course correct is to make this structure visible.
+
+It is not our intention to pontificate. Sermons help no one but we must identify the problem if we hope to discover solutions. We have made efforts. Small things matter --- if we can keep spare toothbrushes in office bathrooms we can surely keep clearly-labeled pads and tampons beside them. So we do. And we can surely have an open and honest conversation about menstruation. So we did. And there is no civilized way to break taboos without a strong menstrual leave policy. So we have one. [5] We have symmetrical maternity and paternity leave because everyone belongs to a family. But these small things should be policies at _every company_. We need more than fairness --- we need equity.
```

Comment:
This is not really a critique of this paragraph itself, but the strength of its link to paragraph 2. Paragraph 2 brings up **structural racism** and **structural casteism** in addition to sexism. However, our efforts are directed towards largely sexism. Is that sensible to be left in-place for a larger point or do you think there's a way to bring about that in this resolving paragraph.

Reply:
Part of my intention was to draw out the different forms of structural discrimination such that structural sexism could be identified in text which is very likely to be read by someone who's never heard that term.

I was hoping to highlight two things:

(1) that structural sexism exists and that nilenso is trying to employ tools (such as affirmative action) to expose and break that structure
(2) that structural sexism follows a similar format to structural casteism and structural racism but is much harder to see, due to its ubiquity

...if I haven't achieved that or if the context seems to dilute the point, I can try pulling those concepts out and focus on sexism exclusively.